llama-index-integrations/indices/llama-index-indices-managed-llama-cloud/llama_index/indices/managed/llama_cloud/base.py

```diff
@@ -57,38 +62,57 @@ class LlamaCloudIndex(BaseManagedIndex):
 
```

Comment:
Can we add a doctoring to the class with example usage?

I actually have no idea what the minimum args to provide is to use this